src/lightkurve/correctors/regressioncorrector.py

```diff
@@ -254,10 +255,14 @@ def correct(
             )
             model = u.Quantity(model, unit=self.lc.flux.unit)
             residuals = self.lc.flux - model
-            # In Astropy>=5.0, residuals will be a MaskedQuantity
-            if hasattr(residuals, 'mask'):
-                residuals = residuals.unmasked
-            self.outlier_mask |= sigma_clip(residuals, sigma=sigma).mask
+            # workaround for https://github.com/astropy/astropy/issues/14360
+            # in passing MaskedQuantity to sigma_clip
+            from astropy.utils.masked import Masked
```

Comment:
Can we have this import at the top of the file? Is there a reason it is in the middle?